### PR TITLE
import into: revert add index via SQL

### PR DESCRIPTION
**Pull request by GMHDBJD**
### What problem does this PR solve?

Issue Number: ref #42930 

Problem Summary:

### What is changed and how it works?
- revert add index via SQL
### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: tests/realtikvtest/importintotest/import_into_test.go [hunk 1]
```diff
@@ -978,128 +978,128 @@ func (s *mockGCSSuite) TestRegisterTask() {
...
-	s.tk.MustQuery("SHOW CREATE TABLE load_data.add_index;").Check(testkit.Rows(
-		"add_index CREATE TABLE `add_index` (\n" +
-			"  `a` int(11) NOT NULL,\n" +
-			"  `b` int(11) DEFAULT NULL,\n" +
-			"  `c` int(11) DEFAULT NULL,\n" +
-			"  PRIMARY KEY (`a`) /*T![clustered_index] CLUSTERED */\n" +
-			") ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin",
-	))
-}
+//func (s *mockGCSSuite) TestAddIndexBySQL() {
```
Comment (D3Hunter) [inline tests/realtikvtest/importintotest/import_into_test.go:981]: can use `s.Skip()`

**Review by D3Hunter — APPROVED** [review]

**Review by lance6716 — APPROVED** [review]

**Review by D3Hunter — APPROVED** [review]

**Review by D3Hunter — APPROVED** [review]

**Review by D3Hunter — APPROVED** [review]

**Pull request merged**